Azrux/Athenas-front
# [feat] - Create DevCard component

## Description
Create Card component for "about us" section
Added a `Test` component to test common ones
Includes `.env` file to `.gitignore`

## Bug
Fixed bug when applying Tailwind classes

## How to test?
Run `npm run dev`
Go to `http://localhost:3000/test`
Check the component is working correctly
Check the responsive design is working correctly
